src/data2img.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import scipy.misc
+import csv
+import os
+
+IMG_SIZE = 48
+csv_file_name = '../data/head.csv'
```

Comment:
yea it should be fer2013.csv - i didnt want to work with the 200mb file here though and created a `head fer2013.csv > head.csv` of it - i see this more of a utility method to be called somewhere else #7 , i guess we are not going to run this file